### PR TITLE
Fix initialize face_engine provider

### PR DESCRIPTION
**Pull request by DeusAnimo**

### CODE REVIEW

**Comment by DeusAnimo** [comment]
waiting for service tests...

File: lunavl/sdk/luna_faces.py [hunk 1]
```diff
@@ -383,24 +383,29 @@ class VLFaceDetector:
...
-        if faceEngine is not None:
-            self.faceEngine = faceEngine
-        else:
-            self.faceEngine = VLFaceEngine()
-        self.estimatorsCollection = FaceEstimatorsCollection(faceEngine=self.faceEngine)
         self._faceDetector: FaceDetector = self.faceEngine.createFaceDetector(detectorType)
         self._estimationSettings: Optional[VLFaceDetectionSettings] = estimationSettings
 
+    @classmethod
+    def initialize(cls, faceEngine: VLFaceEngine) -> None:
```
Comment (matemax) [inline lunavl/sdk/luna_faces.py]: Optional?

File: lunavl/sdk/luna_faces.py [hunk 2]
```diff
@@ -383,24 +383,29 @@ class VLFaceDetector:
     def __init__(
         self,
         detectorType: DetectorType = DetectorType.FACE_DET_DEFAULT,
-        faceEngine: Optional[VLFaceEngine] = None,
```
Comment (matemax) [inline lunavl/sdk/luna_faces.py]: I want set up a custom face Engine for Face Detector instance

File: lunavl/sdk/luna_faces.py (hunk 1 above)
Comment (DeusAnimo) [reply lunavl/sdk/luna_faces.py]: yes

File: lunavl/sdk/luna_faces.py (hunk 2 above)
Comment (DeusAnimo) [reply lunavl/sdk/luna_faces.py]: i did it

File: lunavl/sdk/luna_faces.py [hunk 3]
```diff
@@ -392,13 +392,27 @@ def __init__(
...
             faceEngine: face engine for detector and estimators
+            estimationSettings: settings for detection
         """
-        if faceEngine is not None:
+        if faceEngine is None:
+            if not hasattr(self, "faceEngine"):
+                raise RuntimeError(f"Initialize the '{self.__class__.__name__}' first or pass faceEngine")
+        else:
             self.faceEngine = faceEngine
-            self.estimatorsCollection = FaceEstimatorsCollection(faceEngine=self.faceEngine)
```
Comment (matemax) [inline lunavl/sdk/luna_faces.py:398]: estimatorsCollection  has other faceEngine now. I think that need return  this line

**Pull request merged**